src/content/en/2021/privacy.md

```diff
@@ -5,572 +5,541 @@ description: Privacy chapter of the 2021 Web Almanac covering adoption and impac
 
+Those numbers may seem a small percentage of our 6.2 million websites crawled on desktop, but that opinion changes when segregating the data by site popularity.
 
 {{ figure_markup(
   image="nb_sites_with_cname_tracking_per_rank.png",
-  caption="Number of websites that use CNAME-based tracking according to the their rank.",
-  description="Chart showing the number of websites that use CNAME-based tracking divided by their popularity rank on the desktop client.",
-  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRfOwcNkLp-mYBkmhDV5AFxl8p0ls9oxFnmmo0WUcAJxjxQqmHjquRZCWj_lNZRyFtX3RdH5T92IESu/pubchart?oid=231210750&format=interactive",
+  caption="Number of desktop websites that use CNAME-based tracking according to their rank.",
+  description="Chart showing the number of websites that use CNAME-based tracking divided by their popularity rank on the desktop client. Of the top 1,000 sites, 4.20% use CNAME-based tracking, for the top 10,000 it's 3.33%, for top 10,0000 it's 1.18%, for the top million it's 0.29%, and finally for all sites it's 0.09%.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRfOwcNkLp-mYBkmhDV5AFxl8p0ls9oxFnmmo0WUcAJxjxQqmHjquRZCWj_lNZRyFtX3RdH5T92IESu/pubchart?oid=1885483390&format=interactive",
   sheets_gid="518784663",
   sql_file="nb_sites_with_cname_tracking_per_rank.sql"
   )
 }}
 
+{# TODO - this SQL doesn't exist. Also why only desktop? #}
 
-Apart from the `.com` suffix, a large number of the websites using CNAME-based tracking have a `.jp` domain. Also a notable amount of CNAME trackers are prevalent on `.uk` and `.fr` websites.
+When we look at the rank of the websites that use CNAME-based tracking, we see that 4.20% of the top 1,000 websites embed a CNAME tracker. In the top 10,000, that number falls to 3.33% of websites.
 
 {{ figure_markup(
   image="nb_sites_with_cname_tracking_per_public_suffix.png",
-  caption="Number of websites with public suffix that use CNAME-based tracking",
-  description="Chart showing the number of websites that use CNAME-based tracking on a desktop client, according to the public suffix of the website.",
+  caption="Number of desktop websites with public suffix that use CNAME-based tracking",
+  description="Chart showing the number of websites that use CNAME-based tracking on a desktop client, according to the public suffix of the website. `com` for 6,751 sites, `co.jp` for 656 sites, `org` for 476 sites, `jp` for 363 sites, `com.au` for 297 sites, `ca` for 247 sites, `co.uk` for 188 sites, `de` for 160 sites, `net` for 153 sites, and `fr` for 132.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRfOwcNkLp-mYBkmhDV5AFxl8p0ls9oxFnmmo0WUcAJxjxQqmHjquRZCWj_lNZRyFtX3RdH5T92IESu/pubchart?oid=432551848&format=interactive",
   sheets_gid="1155429637",
   sql_file="nb_sites_with_cname_tracking_per_public_suffix.sql"
   )
 }}
 
+{# TODO - this SQL doesn't exist. Also why desktop? #}
```

Comment:
We are missing the CNAME SQL queries, meaning I couldn't convert to percentage

Reply:
@ydimova